### PR TITLE
add example tests for prepareBaseImage

### PR DESCRIPTION
**Pull request by patrjam**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,13 +24,16 @@
     "image-size": "^0.8.3"
   },
   "devDependencies": {
+    "@wdio/cli": "^6.12.0",
     "allure-commandline": "^2.13.0",
     "codeceptjs": "^3.0.4",
+    "codeceptjs-chai": "^2.2.0",
```
Comment (CDJojn) [inline package.json:30]: Eh.... codeceptjs-assert please

File: steps_file.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// in this file you can append custom step methods to 'I' object
+
+module.exports = function() {
```
Comment (CDJojn) [inline steps_file.js:3]: is this file necessary here?

File: tests/prepareBaseImage-false_test.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
+const DebugCatcher = require('../tools/debugCatcher');
+
+Feature('Global set: prepareBaseImage: false');
+
+Scenario(
```
Comment (CDJojn) [inline tests/prepareBaseImage-false_test.js:5]: what about to have all these RXX test order from 1-18? :D

File: tools/debugCatcher.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+const Helper = require('@codeceptjs/helper');
+
+class DebugCatcher {
+  constructor() {
+    this.messages = '';
+    const orig = Helper.prototype.debug;
```
Comment (CDJojn) [inline tools/debugCatcher.js:6]: why are working with prototype? this is not recommended - debug function is normally accessible by this.debug()

File: tests/prepareBaseImage-false_test.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  'R16 config = false, options = true, baseImageExists = false ',
+  async ({ I }) => {
+    I.say('New base image will be created');
+    const debugCatcher = new DebugCatcher();
```
Comment (CDJojn) [inline tests/prepareBaseImage-false_test.js:9]: Why are you using helper as standard class? Why is helper not instanced in config file by standard way?

**Review by CDJojn — CHANGES_REQUESTED** [review]

File: tests/prepareBaseImage-false_test.js (hunk 5 above)
Comment (CDJojn) [reply tests/prepareBaseImage-false_test.js:9]: Discussed - this is ok this way.

File: tools/debugCatcher.js (hunk 4 above)
Comment (CDJojn) [reply tools/debugCatcher.js:6]: Discussed - this is OK.

File: tests/prepareBaseImage-false_test.js (hunk 3 above)
Comment (CDJojn) [reply tests/prepareBaseImage-false_test.js:5]: Discussed - this order is ok - it is based on base image creating.

**Review by CDJojn — CHANGES_REQUESTED** [review]

**Review by CDJojn — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Trawais) [reply package.json]: replaced

File: steps_file.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+// in this file you can append custom step methods to 'I' object
```
Comment (Trawais) [inline steps_file.js:1]: I believe we don't need this file anymore.

**Review by Trawais — APPROVED** [review]

**Pull request merged**